### DISCUSSION TITLE
Is there a possibility to process <span> somehow instead of just ignoring it

### DISCUSSION THREAD

**Discussion by mos379** [mysticmind/reversemarkdown-net#244] [Q&A]
I have some html which uses <span> tags to separate items and I'd like simply to process them somehow. 
For example just adding a newline after the enclosed text.

Is this somehow possible?

**Comment by mysticmind** [comment]
> I have some html which uses <span> tags to separate items and I'd like simply to process them somehow. 
> For example just adding a newline after the enclosed text.
> 
> Is this somehow possible?

Can you please eloborate and share more details with an example?

**Reply by mos379** [reply]
the following html is what I'm getting for example. 
```html
<p>
	<b>2 sets</b>
	<span style="font-weight: 400;">
</span>
	<span style="font-weight: 400;">:30 mountain climbers</span>
	<span style="font-weight: 400;">
</span>
	<span style="font-weight: 400;">10 ring rows</span>
	<span style="font-weight: 400;">
</span>
	<span style="font-weight: 400;">:30 shoulder taps</span>
	<span style="font-weight: 400;">
</span>
	<span style="font-weight: 400;">5 dive bombers</span>
</p>
```
`<p><b>2 sets</b><span style=\"font-weight: 400;\">\n</span><span style=\"font-weight: 400;\">:30 mountain climbers</span><span style=\"font-weight: 400;\">\n</span><span style=\"font-weight: 400;\">10 ring rows</span><span style=\"font-weight: 400;\">\n</span><span style=\"font-weight: 400;\">:30 shoulder taps</span><span style=\"font-weight: 400;\">\n</span><span style=\"font-weight: 400;\">5 dive bombers</span></p>\n`
the website I'm having the content from is using span items to contain the new lines :( 

the result then is all text without any separation
`**2 sets**:30 mountain climbers10 ring rows:30 shoulder taps5 dive bombers`

**Reply by mysticmind** [reply]
Okay, I am going to spend sometime this week, will have a look.

**Reply by mos379** [reply]
> Okay, I am going to spend sometime this week, will have a look.

Any update on this?

**Reply by mysticmind** [reply]
I am down sick 🤒, hopefully will recover by end of week then will get to these items.

**Reply by mos379** [reply]
Get well soon!

**Reply by mysticmind** [reply]
@mos379 I am having a look at this issue today, will keep you posted.

**Reply by mysticmind** [reply]
I have addressed handling this in v3.21.0. Please get the latest version, check and get back to me.
